### PR TITLE
PR CONTACT FORM

### PR DESCRIPTION
**Pull request by gdvl777**
- Add a contact form to your portfolio and the portfolio of your coding partner.
- Make sure that you guide your users' experience in a way that helps them fill in the form without frustration.

### CODE REVIEW

**Review by shella12 — COMMENTED** [review]
Hi @gdvl777, 

#  Status: Invalidated :red_circle: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

This milestone is a [pair-programming activity](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#work-and-submission-mode) please make sure that everyone has contributed to the project 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/styles.css [hunk 1]
```diff
@@ -0,0 +1,1209 @@
...
+  background: #dfe1e6;
+}
+
+.contact-me {
+  background-color: #6070ff;
+  border: 1px solid #cfd8dc;
+  border-radius: 100px 0 0 0;
+  margin-top: 0%;
+  width: 98%;
+}
```
Comment (levy002) [inline src/styles.css]: - [ ] Kindly make sure the `contact` container has the same background color as the provided design.

- [ ] Kindly make sure your container covers the whole width of the body. You can do that by adjusting your width to `100%`

  - ### YOUR DESIGN
  
chrome-capture-2023-6-11


  - ### FIGMA DESIGN
  
 image

File: src/styles.css [hunk 2]
```diff
@@ -0,0 +1,1209 @@
...
+.bg-image {
+  background-image: url(../assets/contact-me.png);
+  background-repeat: no-repeat;
+  background-position-x: center;
+  background-size: contain;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+}
```
Comment (levy002) [inline src/styles.css]: - [ ] Kindly make sure the background image fits correctly the container in order to match the provided Figma design.

   - ### YOUR DESIGN
   image

   - ### FIGMA DESIGN
   
    image

File: src/styles.css [hunk 3]
```diff
@@ -0,0 +1,1209 @@
...
+  align-items: center;
+  text-align: center;
+  color: #ebebff;
+  justify-content: center;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 28px;
+}
```
Comment (levy002) [inline src/styles.css]: - [ ] Kindly make sure this paragraph is aligned well in order to match the provided Figma design. Please adjust your spacing style to match the layout on the Figma design.


  - ### YOUR DESIGN
   image

  - ### FIGMA DESIGN
   image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @gdvl777   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hello @gdvl777 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- You implemented all suggested changes :heavy_check_mark: 

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**